docs/package_integration/uds-package-practices.md

```diff
@@ -1,110 +1,102 @@
 # UDS Package Practices
 
-This document describes the practices that a UDS package _**must**_, _**should**_ and _**may**_ follow to be considered officially supported as a UDS package.
-
-_and most importantly..._
-> Earning the [<img alt="Made for UDS" src="../made-for-uds.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core) badge and leveraging UDS Core.
+This document describes the standards for [<img alt="Made for UDS" src="../made-for-uds.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core) badging. It is not a comprehensive guide to creating UDS Packages, a more detailed guide can be found [here](guide.md).
 
 > [!NOTE]
-> This badge should link to the `uds-core` repo and should match the height of any other badges (i.e. `20px`).
-
-> [!TIP]
-> This document follows [RFC-2119](https://datatracker.ietf.org/doc/html/rfc2119) for definitions of requirement levels (e.g. **must**, **should** and **may**)
+> This document assumes knowledge of the UDS ecosystem and the UDS Package Custom Resource. If you are unfamiliar with these concepts, please first refer to the [guide](guide.md) providing more detailed information.
 
-Integrating a Package fundamentally means:
-1. Creating a repository `uds-package-<name>`
-2. Integrating the upstream helm chart as a zarf package `zarf.yaml` to build a declarative OCI artifact
-3. Adding a UDS package Custom Resource `uds-package.yaml` to integrate with UDS Core via Pepr
-4. Build a 'zero CVE' package by replacing images with a `*-unicorn` flavored image
+This document outlines services and features of [UDS Core](https://github.com/defenseunicorns/uds-core) UDS package's **must** integrate with, using the [UDS `Package` custom resource](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/README.md#example-uds-package-cr).
 
-This document is intentionally lightweight for quick reference. A more detailed guide can be found [here](guide.md).
+Made for UDS packages can be one of three tiers:
 
-## Integrations
+[<img alt="Gold" src="../made-for-uds-gold.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core), [<img alt="Silver" src="../made-for-uds-silver.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core), or [<img alt="bronze" src="../made-for-uds-bronze.svg" height="20px"/>](https://github.com/defenseunicorns/uds-core).
 
-Below are the minimal services that a UDS package **must** integrate with, using the [UDS `Package` custom resource](https://github.com/defenseunicorns/uds-core/blob/main/src/pepr/operator/README.md#example-uds-package-cr).  These integrations **must** be declarative and automated without requiring click-ops from the person deploying the package.  Packages also **should** consider integrations with any additional UDS services that are relevant to that application.
+> [!IMPORTANT]
+> Packages should aim for Gold by default and only _SETTLE_ for lesser tiers of Bronze and Silver.
```

Comment:
```suggestion
> Packages should aim for Gold by default and only _SETTLE_ for lesser tiers of Bronze and Silver by documented exception.
```

is it worth calling out or linking to examples of documented exceptions here like we did in the bounty program directly? those examples will continue being applicable after the program ends